### PR TITLE
Update ci-e2e.sh to quote LABEL_FILTERS value

### PR DESCRIPTION
**Pull request by thunderboltsid**
This is to ensure the full value of LABEL_FILTERS gets propagated.

### CODE REVIEW

**Review by deepakm-ntnx — APPROVED** [review]

File: scripts/ci-e2e.sh
Comment (adiantum) [inline scripts/ci-e2e.sh:1]: Change also on line 32:
`LABEL_FILTERS="${LABEL_FILTERS}"`

**Review by adiantum — APPROVED** [review]
/lgtm
/approve
/wouf

**Comment by thunderboltsid** [comment]
This works!
```
...
NUTANIX_LOG_LEVEL=debug /home/prow/go/src/github.com/nutanix-cloud-native/cluster-api-provider-nutanix/hack/tools/bin/ginkgo-v2.1.4 -v \
	--trace \
	--progress \
	--tags=e2e \
	--label-filter="!only-for-validation && !cluster-upgrade-conformance && !conformance" \
	--skip=""clusterctl-Upgrade"" \
	--focus="""" \
	--nodes=1 \
	--no-color=false \
	--output-dir="/logs/artifacts" \
	--junit-report="junit.e2e_suite.1.xml" \
	--timeout="24h" \
	--always-emit-ginkgo-writer \
	 ./test/e2e -- \
	-e2e.artifacts-folder="/logs/artifacts" \
	-e2e.config="/home/prow/go/src/github.com/nutanix-cloud-native/cluster-api-provider-nutanix/test/e2e/config/nutanix.yaml" \
	-e2e.skip-resource-cleanup=false \
	-e2e.use-existing-cluster=false
```

**Pull request merged**